ci/playbooks/deploy_watcher_service.yaml

```diff
@@ -8,13 +8,29 @@
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
   tasks:
+    - name: Fetch dlrn md5_hash from master DLRN repo
+      when: fetch_master_dlrn_hash | default(true) | bool
```

Comment:
we should rename `fetch_master_dlrn_hash` var at some point, since if we provide a different `watcher_dlrn_namespace`, we could fetch from a different release too

Reply:
Good point, renamed the var to `fetch_dlrn_hash`.